features/environment.py

```diff
@@ -326,6 +336,15 @@ def from_environ(cls, config) -> "UAClientBehaveConfig":
                 if kwargs[key] == "0":
                     bool_value = False
                 kwargs[key] = bool_value
+
+        if "install_from" in kwargs:
+            install_from = kwargs["install_from"]
+            try:
+                kwargs["install_from"] = InstallationSource(install_from)
+            except ValueError:
+                print("Unknown value for install_from:", install_from)
+                kwargs["install_from"] = InstallationSource.DAILY
```

Comment:
I think we should just raise the `Exception` here and not try to run the tests with a default option